base/sparse/cholmod.jl

```diff
@@ -790,9 +805,19 @@ function sparse{Ti}(A::Sparse{Complex{Float64},Ti}) # Notice! Cannot be type sta
     end
     return convert(Hermitian{Complex{Float64},SparseMatrixCSC{Complex{Float64},Ti}}, A)
 end
-sparse(L::Factor)   = sparse(Sparse(L))
+function sparse(L::Factor)
+    A = Sparse(L)
+    A*A'
```

Comment:
This would have to check if it is an LDLt or an LLt version of `Factor`. I'm sympathetic to changing the behavior of `sparse(Factor)`, but then we'd have to consider the permutations. What is returned from this is still not the same as the input because of them.